content/en/blog/_posts/2025-03-24-swap-fresh-improvements.md

```diff
@@ -1,8 +1,7 @@
 ---
 layout: blog
 title: "Kubernetes 1.32: Fresh Swap Features for Linux Users"
```

Comment:
```suggestion
title: "Swap Features for Linux Users in Kubernetes 1.32"
```

Ah. Because this article is being published outside of the post-release blog series, it's best to give it a new title.

Reply:
Updated